### PR TITLE
feat(ui): TextField

### PR DESCRIPTION
**Pull request by zielu92**
Screenshot 2021-10-16 at 20 11 12
Screenshot 2021-10-16 at 20 11 03
Focused and not focused mode of textfield component

### CODE REVIEW

**Review by Pittawat2542 — CHANGES_REQUESTED** [review]
1. I think you rebase and forget to **force** push again. Hope you do that better next time. Because it makes me hard to review, since all of the files from all previous commits also presents. If you done properly, only your files and your new commit should appear here.
2. I believe you also didn't `git pull --rebase origin main`, since your new commit does not be the last one. (Correct me if I'm wrong)
3. There are still conflict files. Please resolve that.

**Comment by zielu92** [comment]
seems ok now

File: components/ui/TextField.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import styled from "@emotion/styled";
+
+
+const TextField = styled.input`
+  width: 30px;
+  border-radius: 25px;
```
Comment (Pittawat2542) [inline components/ui/TextField.tsx:6]: I believe this is not the correct border radius specify in Figma

File: components/ui/TextField.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  font-size: 16px;
```
Comment (Pittawat2542) [inline components/ui/TextField.tsx:7]: `font-size` is one of a few CSS properties that should use `rem` unit. `1rem` in this case.

File: components/ui/TextField.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  color: #303A52;
```
Comment (Pittawat2542) [inline components/ui/TextField.tsx:8]: Please use color from CSS Custom properties specify in `GlobalStyles`. You can see how to use it online or on other files.

File: .gitignore [hunk 4]
```diff
@@ -34,6 +34,7 @@ yarn-error.log*
 .vercel
 
 
+
```
Comment (Pittawat2542) [inline .gitignore:37]: Remove new line here.

File: .gitignore [hunk 5]
```diff
@@ -65,6 +66,10 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+=======
+# Created by https://www.toptal.com/developers/gitignore/api/webstorm
+# Edit at https://www.toptal.com/developers/gitignore?templates=webstorm
+
```
Comment (Pittawat2542) [inline .gitignore:72]: Remove these lines.

File: .gitignore [hunk 6]
```diff
@@ -171,5 +176,3 @@ fabric.properties
 # CodeStream plugin
 # https://plugins.jetbrains.com/plugin/12206-codestream
 .idea/codestream.xml
-
-# End of https://www.toptal.com/developers/gitignore/api/webstorm,macos
```
Comment (Pittawat2542) [inline .gitignore:175]: These seems to be gone.

**Review by Pittawat2542 — CHANGES_REQUESTED** [review]

**Review by Pittawat2542 — CHANGES_REQUESTED** [review]
`.idea/vcs.xml ` is coming back again 😱.
Maybe add this line to `.gitignore` could help
```
.idea/vcs.xml 
```

**Review by Pittawat2542 — APPROVED** [review]

**Pull request merged**